Docs/Configuration.tex

```diff
@@ -6908,14 +6918,40 @@ \subsection{Audio Properties}\label{uefiaudioprops}
   The quickest way to find the right port is to bruteforce the values from \texttt{0} to \texttt{N - 1}.
 
 \item
+  \texttt{AudioOutRange}\\
```

Comment:
Perhaps this should be a bit mask?

Reply:
(I was trying not to overcomplicate, but it should be.)

Reply:
That will work as long as we have not too many AudioOuts, but I think I have never seen over 63…